libscpi/test/test_parser.c

```diff
@@ -439,6 +440,12 @@ static void testIEEE4882(void) {
     TEST_IEEE4882("STATus:QUEStionable:ENABle 2\r\n", "");
     TEST_IEEE4882_REG(SCPI_REG_QUESE, 2);
 
+    EST_IEEE4882("STATus:QUEStionable:CONDition?\r\n", "0\r\n");
```

Comment:
You have typo in this line `EST` -> `TEST`. See [travis build](https://travis-ci.org/j123b567/scpi-parser/builds/309262791#L1517)